css/desktop.css

```diff
@@ -9,11 +9,11 @@ header {
   box-shadow: 0 0.5px 0 0 #dddfe6;
 }
 
-.nav-menu > a {
+#toggle-menu {
   display: none;
 }
 
-.nav-menu > ul {
+#menu {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/155998151-6de5c8a9-7524-4170-baf1-facd550f8a4e.png)

Therefore:

- [ ] Please, make your menu look like the Figma, including fonts styling, colors, background images, margins.
- [ ] Kindly make the mobile menu span the full height of the screen.
- [ ] Please make your close button smaller, like in Figma.
- [ ] Please, make the margin/padding between the menu items a little more, so it matches the Figma.
- [ ] Please make your font weights for the menu items 400 or "normal", instead of 500.
- [ ] Please, kindly correct the line-height for the elements. and the font-size.
- [ ] Kindly correct the color of the background images.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/155998693-c7d3d541-32ff-4900-9edd-ce72d72db435.png) | ![image](https://user-images.githubusercontent.com/57421823/155997539-c1cbd1dc-7d9f-4a8e-bf09-20d987612d19.png)

That way, your popup will look like the Figma and we will meet this requirement 💪